src/lookups.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  config as bskConfig,
+  validateProofs,
+  resolveZoneFileToProfile,
+} from "blockstack";
+import { validateStacksAddress } from "@stacks/transactions";
+import fetch from "node-fetch";
+//@ts-ignore
+import logger from "winston";
+import * as cheerio from "cheerio";
```

Comment:
cheerio was not included before this pull request can you please revert the change and remove it?